web/src/components/map.js

```diff
@@ -112,7 +112,7 @@ export default class Map {
       nonClickableFill: '#D4D9DE',
       oceanColor: '#FAFAFA',
       co2Scale: {
-        steps: [0, 150, 600, 750],
+        steps: [0, 150, 600, 800],
```

Comment:
why this change?

Reply:
if I remember correctly we want the 750 to have the same color as all values >750, which is different from having the last color set at 800.

Reply:
Ah ok, I see. What I was trying to do there is get rid of the separate constraint constants (in this case `maxCo2`) and make the domain limits be encoded in scales themselves for simplicity.

> we want the 750 to have the same color as all values >750

I made this condition explicitly encoded across all `co2Scale` themes in https://github.com/tmrowco/electricitymap-contrib/pull/2299/commits/79e9082654657f73b50c27098c5cff2a4f995992 - does that make sense?

Reply:
perfect